student-info-system/sis/tests/tests_model.py

```diff
@@ -726,13 +725,14 @@ def test_admin_role(self):
         prof.delete()
         usr.delete()
 
-    def test_admin_excluded(self):
-        usr = User.objects.create(username='foo')
-        prof = usr.profile
-        self.assertEqual(prof.role, Profile.ACCESS_NONE)
-        users = Profile.objects.all()
-        self.assertEqual(users.count(), 1)
-        users_ann = User.annotated()
-        self.assertEqual(users_ann.count(), 0)
-        prof.delete()
-        usr.delete()
+    # test is now invalid given retirement of annotated()
```

Comment:
@bjmckenz this as well?

Reply:
yeah, wanted to commit a reason for it being invalid. Not that important. and it's gone....